src/hardware/tandy_sound.cpp

```diff
@@ -200,8 +139,13 @@ TandyDAC::TandyDAC(const ConfigProfile config_profile, const std::string& filter
 
 	assert(config_profile != ConfigProfile::SoundCardRemoved);
 
+	MIXER_LockMixerThread();
+
 	// Run the audio channel at the mixer's native rate
-	const auto callback = std::bind(&TandyDAC::AudioCallback, this, _1);
+	constexpr bool stereo = false;
+	constexpr bool signeddata = false;
+	constexpr bool nativeorder = true;
```

Comment:
`Stereo`, `SignedData`, etc. by convention for `constexpr`.

Reply:
Renamed